poi-ooxml/src/main/java/org/apache/poi/xssf/streaming/SXSSFRow.java

```diff
@@ -100,15 +103,6 @@ public void setCollapsed(Boolean collapsed) {
         this._collapsed = collapsed;
     }
 //begin of interface implementation
-    /**
-     * {@inheritDoc}
-     */
-    @Override
```

Comment:
I'm not sure we can remove a public method without first deprecating it

Reply:
I've moved it to the `Row` parent interface, so it essentially the same (alias for `cellIterator()`)

Reply:
I see - you can ignore the other comments about removed iterator() methods